kubernetes/provider.go

```diff
@@ -54,11 +54,11 @@ func Provider() *schema.Provider {
 				DefaultFunc: schema.EnvDefaultFunc("KUBE_INSECURE", false),
 				Description: "Whether server should be accessed without verifying the TLS certificate.",
 			},
-			"client_certificate": {
```

Comment:
It looks like by accident you have overwritten the `client_certificate` option with a new one.

Reply:
Fixed. Should be fine now